AnkiDroid/src/main/java/com/ichi2/utils/LanguageUtil.kt

```diff
@@ -42,88 +42,88 @@ object LanguageUtil {
         "Afrikaans" to "af", // Afrikaans
         "አማርኛ" to "am", // Amharic
         "العربية" to "ar", // Arabic
-        "azərbaycan" to "az", // Azerbaijani
-        "беларуская" to "be", // Belarusian
-        "български" to "bg", // Bulgarian
+        "Azərbaycan" to "az", // Azerbaijani
+        "Беларуская" to "be", // Belarusian
+        "Български" to "bg", // Bulgarian
         "বাংলা" to "bn", // Bangla
-        "català" to "ca", // Catalan
+        "Català" to "ca", // Catalan
         "کوردیی ناوەندی" to "ckb", // Central Kurdish
-        "čeština" to "cs", // Czech
-        "dansk" to "da", // Danish
+        "Čeština" to "cs", // Czech
+        "Dansk" to "da", // Danish
         "Deutsch" to "de", // German
         "Ελληνικά" to "el", // Greek
         "English" to "en", // English
-        "esperanto" to "eo", // Esperanto
-        "español (Argentina)" to "es-AR", // Spanish (Argentina)
-        "español (España)" to "es-ES", // Spanish (Spain)
-        "eesti" to "et", // Estonian
-        "euskara" to "eu", // Basque
+        "Esperanto" to "eo", // Esperanto
+        "Español (Argentina)" to "es-AR", // Spanish (Argentina)
+        "Español (España)" to "es-ES", // Spanish (Spain)
```

Comment:
As near as I can tell, Spanish does *not* capitalize the first letter of languages:

- https://www.spanishdict.com/guide/capitalization-in-spanish (find in page "Time Words and Adjectives")
- https://www.tellmeinspanish.com/grammar/spanish-capitalization-rules/#words_that_are_not_capitalized
- https://www.fluentu.com/blog/spanish/spanish-captialization/#toc_16

And of course the most important, the arbiters of the language, the RAE:

https://www.rae.es/buen-uso-espa%C3%B1ol/cu%C3%A1ndo-no-debe-utilizarse-la-may%C3%BAscula-inicial-i

```
• Se inician con minúscula los adjetivos y sustantivos que designan pueblos o etnias, así como los que denotan nacionalidad o procedencia geográfica:

- la cultura mochica, los aztecas, los maoríes, los filipinos.


• También los nombres de las lenguas o idiomas
- El español es la lengua más estudiada del mundo después del inglés.

```
```suggestion
        "español (Argentina)" to "es-AR", // Spanish (Argentina)
        "español (España)" to "es-ES", // Spanish (Spain)
```

Reply:
@mikehardy CLDR appears to disagree

https://github.com/unicode-org/cldr/blob/1eadcf9c1a8dc3c33b001d3d9e98e6c7d9c1e45b/common/main/es.xml#L1243-L1268

truncated:

```xml
	<contextTransforms>
		<contextTransformUsage type="languages">
			<contextTransform type="stand-alone">titlecase-firstword</contextTransform>
			<contextTransform type="uiListOrMenu">titlecase-firstword</contextTransform>
		</contextTransformUsage>
	</contextTransforms>
```